src/mozanalysis/experiment.py

```diff
@@ -625,7 +625,7 @@ def _build_enrollments_query_normandy(self, time_limits: TimeLimits) -> str:
         return """
         SELECT
             e.client_id,
-            `mozfun.map.get_key`(e.event_map_values, 'branch')
+            `mozfun.map.get_key`(e.event_map_values, 'branchSlug')
```

Comment:
Should this change be applied on line 735 instead? This one looks at enrollments